Week-5/Ex2.js

```diff
@@ -0,0 +1,23 @@
+//input parameter c
+//purpose of function is to check if given char is a vowel or not.
+// return true or false.
+function isVowel(c) {
+    return "aeiou".includes(c)
+}
+
+function vowelCount(str) {
+    const vCount = new Map();
+    for(let c of str) {
+        c = c.toLowerCase()
+        if(isVowel(c)) {
+            if(vCount.has(c)) {
+                vCount.set(c, vCount.get(c)+1);
```

Comment:
We can use ternary operator here

Reply:
Also try to use optional chaining as a habit for null handling.
https://www.javascripttutorial.net/es-next/javascript-nullish-coalescing-operator/